db/seeds.rb

```diff
@@ -41,19 +41,19 @@
                      brand: 'Ride',
                      image: File.open('app/assets/images/holman_bib.png')
                    },
-                   { name: 'GOOD VIBES ZIP HOODIE',
+                   { name: 'ZIP HOODIE',
                      description: 'Features: 80% cotton, 20% polyester Colors: Heather Gray, Olive Sizes: S, M, L, XL',
                      price: 60,
                      brand: 'K2',
                      image: File.open('app/assets/images/good_vibes_hoody.png')
                    },
-                   { name: '[ak] Leather Tech Glove',
+                   { name: '[AK] Tech Glove',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.